spec/commands/asset/link_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'alces_utils'
+require 'cache/asset'
+require 'filesystem'
+
+RSpec.describe Metalware::Commands::Asset::Link do
+  include AlcesUtils
+
+  let :asset_name { 'asset_test' }
+  let :node_name { 'test_node' }
+  let :node { alces.nodes.find_by_name(node_name) }
+  let :content { { node: { node_name.to_sym => asset_name } } }
+
+  AlcesUtils.mock(self, :each) do
+    mock_node(node_name)
+  end
+
+  def run_command
+    Metalware::Utils.run_command(described_class,
+                                 asset_name,
+                                 node_name,
+                                 stderr: StringIO.new)
+  end
+
+  it 'errors when the asset does not exist' do
```

Comment:
We have a few different commands that are meant to error if the asset does not exist.

This should be extracted to a `shared_example` which will clean up the specs.